plugins/processors/awsappsignals/processor.go

```diff
@@ -132,6 +136,7 @@ func (ap *awsappsignalsprocessor) processMetrics(ctx context.Context, md pmetric
 			metrics := ils.Metrics()
 			for k := 0; k < metrics.Len(); k++ {
 				m := metrics.At(k)
+				m.SetName(metricCaser.String(m.Name())) // Ensure metric name is uppercase
```

Comment:
Nitpick - comment should say its in sentence case, not upper case

Reply:
+1